### PR TITLE
Python 2.7 Compatibility

### PR DESCRIPTION
**Pull request by lilleswing**
The tests were not being run with python 2.7, but rather with 3.5 twice.  Fix the test running and python 2.7 incompatibilities.

### CODE REVIEW

File: deepchem/dock/tests/test_binding_pocket.py [hunk 1]
```diff
@@ -149,35 +149,6 @@ def test_convex_find_pockets(self):
 
     assert len(pockets) < len(all_pockets)
 
-  def test_rf_convex_find_pockets(self):
```
Comment (rbharath) [inline deepchem/dock/tests/test_binding_pocket.py]: Should this test be removed?

File: deepchem/dock/tests/test_docking.py [hunk 2]
```diff
@@ -82,25 +82,6 @@ def test_vina_grid_rf_docker_specified_pocket(self):
     # Check returned files exist
     assert score.shape == (1,)
 
-  def test_pocket_vina_grid_rf_docker_dock(self):
```
Comment (rbharath) [inline deepchem/dock/tests/test_docking.py]: Same question here.

File: deepchem/dock/tests/test_binding_pocket.py (hunk 1 above)
Comment (lilleswing) [inline deepchem/dock/tests/test_binding_pocket.py]: So if you look down a couple of lines they were only run w/ python 2.7 which means never.  They are based off of a function which throws a ValueError("Not Implemented").  I can mark the tests as ignored?

Comment (rbharath) [inline deepchem/dock/tests/test_binding_pocket.py]: Ah, good find. Marking the tests as ignored seems like a good intermediate. There's been some good work on binding site prediction [recently](https://academic.oup.com/bioinformatics/article-abstract/doi/10.1093/bioinformatics/btx350/3859178/DeepSite-Protein-binding-site-predictor-using-3D?redirectedFrom=fulltext) which I'd like to get into DeepChem. Keeping these older tests around will be useful, even if we need to fix them up later.

Comment (lilleswing) [inline deepchem/dock/tests/test_binding_pocket.py]: yeah paper is behind a paywall :( We had to ask the authors for a preprint

Comment (lilleswing) [inline deepchem/dock/tests/test_binding_pocket.py]: I'll mark as ignored

**Comment by rbharath** [comment]
LGTM.

**Comment by rbharath** [comment]
I think this PR might have broken the build. Going to revert for now...

**Comment by rbharath** [comment]
Never mind; reverting is trickier than I thought.

@lilleswing Would you mind taking a look?

**Comment by lilleswing** [comment]
Yeah looking now..... not sure why it wasn't caught earlier by travis.

**Pull request merged**